src/com/rcv/Main.java

```diff
@@ -47,33 +49,59 @@ public static void main(String[] args) {
   static ElectionConfig loadElectionConfig(String configPath) {
     // config: the new object
     ElectionConfig config = null;
-    try {
-      // rawConfig holds the basic election config data parsed from json
-      RawElectionConfig rawConfig;
-      // parse raw config
-      rawConfig = JsonParser.parseObjectFromFile(configPath, RawElectionConfig.class);
-      // create and validate new ElectionConfig
-      if (rawConfig != null) {
-        config = new ElectionConfig(rawConfig);
-        // set up log output
-        Logger.setup(config.auditOutput());
+
+    // hitError tracks whether anything went wrong during loading
+    boolean hitError = false;
+
```

Comment:
"error" would be clearer

Reply:
I disagree -- if it were called `error`, I would expect it to be a string or something with info about the error. I changed it to `encounteredError` to make it even clearer that it's a boolean.